core/utils/silly-i18n/src/index.js

```diff
@@ -19,7 +13,7 @@ export const determineLanguage = () =>
  */
 export class I18n {
     /**
-     * Class constructor
+     * Class constructor. The locale used will be auto-detected, and stored as a private-ish attribute.
```

Comment:
Not a huge fan of this - I'd add a `locale` parameter, so we don't need to mess around with a private in the tests, not magically set the automatically determined locale here. I'm not opposed to making this the default behaviour though.

Reply:
The comment there could still use an update, since it is now an explicit parameter with a default value. Seems you can just restore this line to `* Class constructor`, since it's documented below.